src/diffusers/models/attention.py

```diff
@@ -221,11 +221,15 @@ def forward(self, hidden_states, encoder_hidden_states=None, timestep=None, retu
         # 3. Output
         if self.is_input_continuous:
             if not self.use_linear_projection:
-                hidden_states = hidden_states.reshape(batch, height, weight, inner_dim).permute(0, 3, 1, 2)
+                hidden_states = (
+                    hidden_states.reshape(batch, height, weight, inner_dim).permute(0, 3, 1, 2).contiguous()
```

Comment:
Shouldn't we do the contiguous on `mps` only?

Reply:
I believe everywhere else where we reshape and permute we also make contiguous... so it follows that convention

Reply:
In particular in the VQ-VAE and VAE models...